subworkflows/local/interaction_caller/maps_peak.nf

```diff
@@ -0,0 +1,57 @@
+/*
+ * Call interaction peaks by MAPS
+ */
+
+include { MAPS_MAPS             } from '../../../modules/local/maps/maps'
+include { MAPS_CALLPEAK         } from '../../../modules/local/maps/callpeak'
+include { READS_SUMMARY
+    as MAPS_STATS               } from '../../../modules/local/reads_summary'
+include { MAPS_REFORMAT         } from '../../../modules/local/maps/reformat'
+include { MAPS_RAW2BG2          } from '../../../modules/local/maps/raw2bg2'
+include { COOLER_LOAD           } from '../../../modules/local/cooler/load/main'
+include { COOLER_MERGE          } from '../../../modules/nf-core/cooler/merge/main'
+include { COOLER_ZOOMIFY
+    as COOLER_ZOOMIFY_MAPS      } from '../../../modules/nf-core/cooler/zoomify/main'
+
+workflow MAPS_PEAK {
+    take:
+    reads                     // channel: [ meta, bin_size, path(macs2), path(long_bedpe), path(short_bed), path(background) ]
+    chromsizes                // channel: [ path(chromsizes) ]
+
+    main:
+    //create parameter table
+    //input=val(meta), val(bin_size), path(macs2), path(long_bedpe), path(short_bed), path(background)
+    //maps from bedpe
+    ch_version = MAPS_MAPS(reads, params.long_bedpe_postfix, params.short_bed_postfix).versions
```

Comment:
It would be more clear if we first run the module and then assign the version to the channel.